### ISSUE TITLE
Redirect input

### ISSUE THREAD

**Issue by EderBeldad** [Shatur/neovim-tasks#15]
Can't find a way to redirect input by using the configuration files. Is there a way?

I tried something like: 

```json
{
    "cmake": {
        "cmd": "cmake",
        "dap_name": "lldb",
        "target": "longest-palindrome",
        "build_dir": "build",
        "args": {
            "configure": ["-DCMAKE_EXPORT_COMPILE_COMMANDS=1"],
            "debug": ["<", "./sample-input.txt"]
        },
        "build_type": "Debug"
    }
}
```

**Comment by Shatur** [comment]
>             "debug": ["<", "./sample-input.txt"]

Should it be `"debug": [">", "./sample-input.txt"]`?

**Comment by EderBeldad** [comment]
> Should it be `"debug": [">", "./sample-input.txt"]`?

That would redirect output, not input (and would also be a desirable feature!).

Ideally (imho) it should work as [vscode](https://code.visualstudio.com/docs/editor/debugging#_redirect-inputoutput-tofrom-the-debug-target). Although having stdin/stdout as extra parameters would also be convenient.

**Comment by Shatur** [comment]
Oh, I see. Makes sense, but not sure if it's possible.
I use Job API from plenary: https://github.com/nvim-lua/plenary.nvim/blob/master/lua/plenary/job.lua
If it does not support it, I can't fix it. I would suggest to open issue there.

**Comment by Shatur** [comment]
Right, then we probably could add support for it by providing additional parameter with paths.
And just use it in this function:
https://github.com/Shatur/neovim-tasks/blob/f42b164499b6e833ac6b94993abc215fc4a0db17/lua/tasks/runner.lua#L98

I currently busy at the moment, but if you would like to play with it - PRs are welcome.
